lib/phases/cold-launch.js

```diff
@@ -116,15 +119,17 @@ ColdLaunch.prototype.capture = function() {
   this.dispatcher.on(Phase.MEMORYENTRY, function(entry) {
     phase.debugEventEntry(Phase.MEMORYENTRY, entry);
 
-    if (entry.context !== app) {
```

Comment:
Same rule as above: error Assignment to function parameter 'entry' no-param-reassign